packages/vx-grid/src/grids/GridColumns.tsx

```diff
@@ -5,11 +5,18 @@ import { Group } from '@vx/group';
 import { Point } from '@vx/point';
 import { Scale, CommonGridProps } from '../types';
 
+export type GridColumnLineProps<ScaleInput> = (
+  tick: ScaleInput,
+  index: number
```

Comment:
it could be useful to pass `ticks` as the third argument here so you can more easily know the exact number of ticks (say if you want to style the last one). we do this for [`tickFormat`](https://github.com/airbnb/visx/blob/master/packages/visx-axis/src/types.ts#L19) in `@visx/axis` and it's useful because `numTicks` is approximate / can be difficult to know final number of ticks ahead of time.

Reply:
also, I'm wondering if we can simply add this to `CommonGridProps` as `gridLineProps`, and then in `Grid.tsx` we update the name to be specific to column / row (`gridColumnLineProps/gridRowLineProps`). This more closely matches what we do for other props in `Grid` and I think there's no reason it needs to be specific on the `GridRow/Column` components. It will require passing the `ScaleInput` generic to `CommonGridProps`, tho.